controllers/numaresourcesoperator_controller.go

```diff
@@ -305,6 +311,53 @@ func (r *NUMAResourcesOperatorReconciler) syncNUMAResourcesOperatorResources(ctx
 	return daemonSetsNName, nil
 }
 
+func (r *NUMAResourcesOperatorReconciler) deleteUnusedDaemonSets(ctx context.Context, instance *nropv1alpha1.NUMAResourcesOperator, mcps []*machineconfigv1.MachineConfigPool) error {
+	klog.Info("Delete DS start")
+	var daemonSetList appsv1.DaemonSetList
+	if err := r.List(ctx, &daemonSetList, &client.ListOptions{Namespace: instance.Namespace}); err != nil {
+		klog.Infof("error while getting DS list: %v", err)
```

Comment:
`ErrorS` seems better here

Reply:
> it seems we have still unanswered comments? perhaps a new version wasn't pushed?

Sorry forgot to change the "ErrorS" comments. 
I think all comments, but the one regarding the convenience to use a list of errors or not, have been addressed now.

Reply:
fixed?